libs/ledger-live-common/src/families/bitcoin/wallet-btc/wallet.ts

```diff
@@ -144,13 +148,24 @@ class BitcoinLikeWallet {
         }
       }
     });
+
+    const outputScripts = outputAddresses.map((addr) =>
+      account.xpub.crypto.toOutputScript(addr)
+    );
+
+    if (opReturnData) {
+      outputScripts.push(
+        account.xpub.crypto.toOpReturnOutputScript(opReturnData)
+      );
+    }
+
     // fees if we use all utxo
     const fees =
       feePerByte *
       utils.maxTxSizeCeil(
         usableUtxoCount,
-        outputAddresses,
-        outputAddresses.length == 0,
+        outputScripts,
+        outputScripts.length == 0,
```

Comment:
maybe not directly related to this PR. But I think we can
change 
```
outputScripts.length == 0,
```
to
```
false
```
because if we use all utxo, we don't have change by any mean

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2278

Reply:
yes, you are right. I've approved your #2278 
I just checked the flow in LLM and LLD. 
outputScripts.length == 0
should be always false, because in LLD/LLM we always calculate the max spendable after user inputs destination address.
The current PR and #2278 are completely transparent for LLM/LLD users. In my opinion, We can merge two PRs in develop and QA will have non-reg test for bitcoin sending and bitcoin send max before the release. I don't think QA need to test #2278 independently as it is not a feature or bug. It is just a code cleanup.